src/components/PopupTrigger.js

```diff
@@ -144,6 +144,7 @@ export const MenuTrigger = ({ children, content, popupProps = {}, ...props }) =>
     content: h(VerticalNavigation, [content]),
     popupProps: {
       role: 'menu',
+      'aria-modal': undefined,
```

Comment:
This was causing a failure because PopupTrigger (line 143) sets aria-modal to true, but the menu role does not support aria-modal.